pkg/scheduler/extender/extender.go

```diff
@@ -627,6 +658,42 @@ func createRequestFromPVCSpec(volumeName, storageType string, resourceRequiremen
 	}
 }
 
+func calculateScheduleTime(ctx context.Context, e *Extender, namespace string, podName string) (float64, float64, error) {
+	// record schedule information to metrics
+	cs, err := clientset.GetK8SClientset()
+	if err != nil {
+		e.logger.Errorf("cannot get clientset: %v", err)
+		return 0, 0, err
+	}
+	events, err := cs.CoreV1().Events(namespace).List(ctx, metav1.ListOptions{FieldSelector: "involvedObject.name=" + podName, TypeMeta: metav1.TypeMeta{Kind: "Pod"}})
+	if err != nil {
+		e.logger.Errorf("cannot read events for pod %v: %v", podName, err)
+		return 0, 0, err
+	}
+
+	return calculate(events.Items)
+}
+
+func calculate(events []coreV1.Event) (float64, float64, error) {
+	var totalTime, sinceLastTime float64
+	const Killing = "Killing"
```

Comment:
it's better to define the const in a common file.  so that the latter test code can re-use it. 
otherwise, it is easy to make mistakes when changing code next time.

Reply:
yes, I think put it into metrics.go is better to reuse it, thanks for your advice.